packages/client/src/components/inspector/InspectorDataField/Actions.vue

```diff
@@ -118,7 +119,7 @@ function quickEditNum(v: number | string, offset: 1 | -1) {
       </template>
     </template>
     <!-- delete prop, only appear if depth > 0 -->
-    <VueButton v-if="!props.disableEdit && depth > 0" v-bind="iconButtonProps" :class="buttonClass" @click.stop="quickEdit(rawValue, true)">
+    <VueButton v-if="!props.disableEdit && depth > 0 && props.rootId !== 'getters'" v-bind="iconButtonProps" :class="buttonClass" @click.stop="quickEdit(rawValue, true)">
```

Comment:
`rootId: 'getters'` is not exclusively used by Pinia.

You can approach like passing `{ disableEdit: true }` from props in Pinia's getter field.

Reply:
I have improved the code and thank you very much for your help. If you could help me check again, I would greatly appreciate it
@Azurewarth0920

Reply:
@konata33 
I think you can pass the `disable-edit` to component `InspectorState` (https://github.com/vuejs/devtools-next/blob/main/packages/client/src/pages/pinia.vue#L77) in getter field.